packages/safe-core-sdk/src/contracts/MultiSend/MultiSendEthersContract.ts

```diff
@@ -0,0 +1,20 @@
+import { MultiSend as MultiSend_V1_1_1 } from '../../../typechain/src/ethers-v5/v1.1.1/MultiSend'
+import {
+  MultiSend as MultiSend_V1_3_0,
+  MultiSendInterface
+} from '../../../typechain/src/ethers-v5/v1.3.0/MultiSend'
+import MultiSendContract from './MultiSendContract'
+
+abstract class MultiSendEthersContract implements MultiSendContract {
+  constructor(public contract: MultiSend_V1_1_1 | MultiSend_V1_3_0) {}
+
+  getAddress(): string {
+    return this.contract.address
+  }
+
+  encode: MultiSendInterface['encodeFunctionData'] = (methodName: any, params: any): string => {
```

Comment:
Is it safe to use `any`?

Reply:
Params depend on the method name and both of them are defined in an auto-generated file.
For now it is safe as it is only used internally and the all the uses of this function are tested.
I can look into this later